commands/bake.go

```diff
@@ -144,7 +145,13 @@ func runBake(dockerCli command.Cli, targets []string, in bakeOptions, cFlags com
 	}()
 
 	if url != "" {
-		files, inp, err = bake.ReadRemoteFiles(ctx, nodes, url, in.files, printer)
+		localFilesArg, remoteFilesArg := getFilesArgs(in.files)
+		files, inp, err = bake.ReadRemoteFiles(ctx, nodes, url, remoteFilesArg, printer)
+		if len(localFilesArg) > 0 {
+			var localFiles []bake.File
+			localFiles, err = bake.ReadLocalFiles(localFilesArg, dockerCli.In())
+			files = append(localFiles, files...)
+		}
 	} else {
 		files, err = bake.ReadLocalFiles(in.files, dockerCli.In())
```

Comment:
We should make sure that `cwd://` is still valid for local files - even though it'll just be stripped.

We can move the `getFilesArgs` call out of the branch loop and just call `ReadLocalFiles` on both local and remote splits.

Reply:
>just call ReadLocalFiles on both local and remote splits.

@jedevc are you sure about that? don't we need 2 different calls do either bake.ReadRemoteFiles or bake.ReadLocalFiles?

Reply:
Aha, yeah you got the gist of what I meant, thanks :)